### ISSUE TITLE
Want to create another iscsi target in gwcli

### ISSUE THREAD

**Issue by sadiq1425** [ceph/ceph-iscsi-cli#104]
/iscsi-target> create iqn.2003-01.com.redhat.iscsi-gw:ocp-igw
Only ONE iscsi target image is supported
/iscsi-target> exit

Hi 
I am getting the above error while createing another target in 2 node iscsi gateway setup . 
My use case is for RHEV Self hosted engine where it is recommended to have two different targets , one for self hosted neginer and another for the additional storage domains . 

Can you please suggest how i can accomplish this with Ceph Iscsi gateway since i dont have another storage option 

Thanks
Sadiq

**Comment by dillaman** [comment]
Only a single target is supported, but you can map different LUNs (RBD images) to different hosts/host groups.

**Comment by sadiq1425** [comment]
i am not ablet o create host-group . Is it supported . 
I get thsi error when addign a client in host-groups

Failed :hostgroup create/update failed on dn-ceph-eaa. Group request failed validation 

thanks
Sadiq

**Comment by dillaman** [comment]
Yes, it should work. Can you check "/var/log/rbd-target-api.log" to look for error messages? What gwcli command did you execute that failed? adding disks or adding hosts?
